cue/common/datasource.cue

```diff
@@ -14,8 +14,10 @@
 package common
 
 #datasourceSelector: {
-	datasource?: {
+	datasource?: =~#variableSyntaxRegex | {
 		kind:  string
 		name?: string
 	}
 }
+
+#variableSyntaxRegex: "^\\$\\w+$"
```

Comment:
```suggestion
```
You should be able to remove this actually, this file and `variable.cue` are both part of the same package and `variableSyntaxRegex` is already declared there

Reply:
good catch. removed in 97a54b87b594dda3fcf13781aee4f0e2d945066a